packages/ckeditor5/src/plugin.js

```diff
@@ -257,6 +257,14 @@ export default class MathType extends Plugin {
       // And obtain the complete formula
       formula = Util.htmlSanitize(`<math${mathAttributes}>${formula}</math>`);
 
+      // Replaces the < & > characters to its HTMLEntity to avoid render issues.
+      formula = formula.split('"<"').join('"&lt;"')
+        .split('">"')
+        .join('"&gt;"')
+        .split('><<')
+        .join('>&lt;<');
```

Comment:
Where does this case show up? The other two make sense as the attributes `open="<"` etc. But this last one seems more convoluted.

Reply:
There are cases where the character inside the mathml can be the character `<` as a part of the formula. It also follows the logic of the viewer